utils/zerovec/src/map/borrowed.rs

```diff
@@ -73,6 +73,8 @@ impl<'a, K, V> Default for ZeroMapBorrowed<'a, K, V>
 where
     K: ZeroMapKV<'a>,
     V: ZeroMapKV<'a>,
+    <<K as ZeroMapKV<'a>>::Container as ZeroVecLike<K>>::BorrowedVariant: 'static,
+    <<V as ZeroMapKV<'a>>::Container as ZeroVecLike<V>>::BorrowedVariant: 'static,
```

Comment:
I added these bounds to the function that needs them, but I could have added it to the associated type. However, adding it to the associated type requires `T: 'static` which is not something we currently have as a blanket requirement.